app/models/report.rb

```diff
@@ -222,6 +233,25 @@ def report_data
     docs_to_records(@response.documents)
   end
 
+  ##
+  # Converts the `report_data` into CSV data
+  #
+  # @return [Enumerator] data in CSV format
+  def to_csv
```

Comment:
wouldn't "csv" be more idiomatic ruby?

Reply:
I don't think so, no.`to_{format}` methods are quite common, e.g., `to_json`, `to_h`, `to_a`, `to_s`,